packages/framework/tinylicious-client/README.md

```diff
@@ -25,25 +28,29 @@ const tinyliciousClient = new TinyliciousClient(config);
 
 ## Fluid Containers
 
-A Container instance is a organizational unit within Fluid. Each Container instance has a connection to the defined Fluid Service and contains an independent collection of collaborative objects.
+A Container instance is a organizational unit within Fluid. Each Container instance has a connection to the defined
+Fluid Service and contains an independent collection of collaborative objects.
 
-Containers are created and identified by unique ids. Management and storage of these ideas are the responsibility of the developer.
+Containers are created and identified by unique ids. Management and storage of these ideas are the responsibility of
+the developer.
 
 ## Using Fluid Containers
 
-Using the default `TinyliciousClient` object the developer can create and get Fluid containers. Because Fluid needs to be connected to a server containers need to be created and retrieved asynchronously.
+Using the default `TinyliciousClient` object the developer can create and get Fluid containers. Because Fluid needs to
+be connected to a server containers need to be created and retrieved asynchronously.
 
 ```javascript
 import TinyliciousClient from "@fluidframework/tinylicious-client";
 
 const tinyliciousClient = new TinyliciousClient(config);
-await tinyliciousClient.createContainer( { id: "_unique-id_" }, /* schema */);
-const { fluidContainer, containerServices } = await tinyliciousClient.getContainer({ id: "_unique-id_" }, /* schema */);
+await tinyliciousClient.createContainer(schema);
+const { fluidContainer, containerServices } = await tinyliciousClient.getContainer("_unique-id_", schema);
```

Comment:
```suggestion
const { fluidContainer, containerServices } = await tinyliciousClient.getContainer("_unique-id_", /* schema */);
```

Reply:
Why do we need to wrap it as a comment? Is it optional?

Reply:
Just since schema is defined in the next section. Alternatively, we can have "Defining Fluid Containers" section come first and "Using Fluid Containers" come next and have the schema without commenting

Reply:
done.